unit-tests/live/config/test-eth-config.py

```diff
@@ -0,0 +1,146 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2025 Intel Corporation. All Rights Reserved.
+
+# Currently only D555 supports DDS configuration natively
+# test:device each(D555)
+
+import pyrealsense2 as rs
+import pyrsutils as rsutils
+from rspy import test
+
+
+get_eth_config_opcode = 0xBB
+set_eth_config_opcode = 0xBA
+
+default_values_param = 0
+current_values_param = 1
+
+
+ctx = rs.context()
+dev = ctx.query_devices()[0]
```

Comment:
please use the get_first_device_or_exit function

Reply:
Done